### PR TITLE
Make suppress jquery opt-in

### PR DESCRIPTION
**Pull request by rondale-sc**

### CODE REVIEW

**Comment by rondale-sc** [comment]
This should address some of the concerns (though admittedly not all) mentioned in #54 

/cc @fivetanley @rwjblue

File: README.md [hunk 1]
```diff
@@ -121,6 +121,26 @@ rebuild when an EmberCLI file is changed. The `--output-path` flag specifies
...
+If you have the `jquery-rails` gem installed into your rails app and included it
+onto the page you ember app is rendered to it might be beneficial to prevent
+EmberCLI from packaging its own jquery.  This can reduce the size of your Ember
+related files.
+
+In order to do this simply set `suppress_jquery: true` in your initializer:
+
+```ruby
+EmberCLI.configure do |c|
+  c.app :frontend, supress_jquery: true
```
Comment (fivetanley) [inline README.md]: suppress has two p's

Comment (rondale-sc) [inline README.md]: Quite right.

**Comment by fivetanley** [comment]
LGTM! Woo! Thanks!

**Comment by rwz** [comment]
//cc @fivetanley

**Comment by rondale-sc** [comment]
@fivetanley Pavel and I worked out a little bit more robust of a solution.  But it should work the same.  :)  The readme has all the deets.

**Comment by fivetanley** [comment]
looks awesome to me!

**Comment by fivetanley** [comment]
<3 thanks so much!

**Comment by rondale-sc** [comment]
np, keep the issues comin' :)))))

**Pull request merged**